From bc99d2ae5c4ed44e55a796443759c0ff0f97fd6f Mon Sep 17 00:00:00 2001
From: Delphox <smugleaf@live.com>
Date: Sat, 7 Aug 2021 22:23:24 -0300
Subject: [PATCH 195/506] Fix empty Edge extension ID on href (#294)

* Fix empty Edge extension ID on href

* remove double space
---
 .../src/org/chromium/chrome/browser/PersonalizeResults.java     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/PersonalizeResults.java b/chrome/android/java/src/org/chromium/chrome/browser/PersonalizeResults.java
index efb80c5b..621ea385 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/PersonalizeResults.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/PersonalizeResults.java
@@ -158,7 +158,7 @@ public class PersonalizeResults {
 +"            individualButton.style.background = 'rgb(0, 120, 212) !important';"
 +"            individualButton.style.height = '60px';"
 +"            individualButton.removeAttribute('disabled');"
-+"            individualButton.innerHTML = \"<a href='https://edge.microsoft.com/extensionwebstorebase/v1/crx?response=redirect&acceptformat=crx3&x=id%3D' + individualButton.id.split('-')[1] + '%26installsource%3Dondemand%26uc' target='_blank' style='color: white; text-decoration: none'><b>Get CRX</b><br>(Hold and tap<br>Download Link)</a>\";"
++"            individualButton.innerHTML = \"<a href=https://edge.microsoft.com/extensionwebstorebase/v1/crx?response=redirect&acceptformat=crx3&x=id%3D\" + individualButton.id.split('-')[1] + \"%26installsource%3Dondemand%26uc target='_blank' style='color: white; text-decoration: none'><b>Get CRX</b><br>(Hold and tap<br>Download Link)</a>\";"
 +"        });"
 +"    });"
 +"})();";
